packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1035,6 +1041,13 @@ export class GarbageCollector implements IGarbageCollector {
 			return undefined;
 		}
 
+		// Add the options that are used to run GC to the telemetry context.
+		telemetryContext?.set(
+			"fluid_GC",
+			"Options",
+			JSON.stringify({ fullGC, runSweep: options.runSweep }),
+		);
+
```

Comment:
nit. Have you considered simply having the "fluid_Summarize " event and add to the context the additional info like summaryStateNeedsReset, etc. ?  It would be nice to have a single telemetry event.

Reply:
Not sure I follow. Are you saying have a new event "fluid_Summarize" and add these pproperties plus additional to that event?

Reply:
No - disregard - looks good.